### ISSUE TITLE
IP:9999 opens with TasmoAdmin/tasmoadmin/index.php

### ISSUE THREAD

**Issue by Gisbert1** [TasmoAdmin/TasmoAdmin#441]
I moved to Debian bullseye today, but not sure if the following is a result from this or just independent.
When I type into my brower _IP-address server:9999_ the content of TasmoAdmin/tasmoadmin/index.php is given as response, but not the expected loging page or anything else which a user wants to see.

Any ideas what I did wrong?
I followed the instructions from: [https://github.com/reloxx13/TasmoAdmin/wiki/Guide-for-Ubuntu-Server-16.04](url)

**Comment by xxstasxx** [comment]
same issue, do you have a solution?

**Comment by chicos04** [comment]
Hi, I've just updated to Truenas 13 so there is no longer a native plugin that I can get working. So I've built a VM with Ubuntu and followed the same instructions as above. I too get the same error. Issue is that as its the first time accessing this new install I don't have the web page to "Activate Login".

I've checked the /src/Config.php which I gather is the initial way of bootstrapping this into creating a MyConfig.json file and that looks all good. Currently there is no MyConfig.json file in existence as I was expecting the first launch to create it.

I've created one from my ond install (with password removed) but still no luck.

Any help appreciated.
